Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM circleci/php:7.2-cli-node
 
+# Install libpng-dev required by some frontend libs
+RUN apt-get update && apt-get install -y libpng-dev
```

Comment:
The `apt-get update` is not needed, Docker takes care of that. Since the base image we use switches to the `circleci` user, we need to use `sudo` to install packages.

Also, if webpack uses this library to manage images, I would explicitly mention it here.